fix: add frequently asked questions to auth identity linking docs (#31016)
supabase/supabase: apps/docs/content/guides/auth/auth-identity-linking.mdx

</TabPanel>
</Tabs>

## Frequently asked questions

### How to add email/password login to an OAuth account?
Call the `updateUser({ password: 'validpassword'})` to add email with password authentication to an account created with an OAuth provider (Google, GitHub, etc.).

> ### What happens if I sign up with email after OAuth?

Author: ```suggestion
### Can you sign up with email if already using OAuth?
```
